File: src/main/java/com/chattriggers/ctjs/launch/mixins/transformers/NbtCompoundAccessor.java
package com.chattriggers.ctjs.launch.mixins.transformers;

import net.minecraft.nbt.NbtCompound;
import net.minecraft.nbt.NbtElement;
import org.spongepowered.asm.mixin.Mixin;
import org.spongepowered.asm.mixin.gen.Accessor;

import java.util.Map;

@Mixin(NbtCompound.class)
public interface NbtCompoundAccessor {
    @Accessor
    public Map<String, NbtElement> getEntries();
}

File: src/main/kotlin/com/chattriggers/ctjs/launch/mixins/transformers/NbtCompoundAccessor.kt
package com.chattriggers.ctjs.launch.mixins.transformers

import net.minecraft.nbt.NbtElement
import org.spongepowered.asm.mixin.Mixin
import org.spongepowered.asm.mixin.gen.Accessor

@Mixin(NbtCompoundAccessor::class)
interface NbtCompoundAccessor {
    @Accessor
    fun getEntries(): MutableMap<String, NbtElement>
}
